styleguide/source/assets/scss/01-atoms/_table.scss

```diff
@@ -1,116 +1,92 @@
-/**
- * @file
- * Define responsive table.
- *
- *
+/*
+ * jQuery Basic Table
+ * Author: Jerry Low
  */
 
 table {
-  display: block;
-  width: 100%;
+  background: $white;
   border-collapse: collapse;
-  border-spacing: 0;
-  border: 1px solid $gray-64;
-  table-layout: fixed;
+  width: 100%;
+}
 
-  @include breakpoint($bp-small) {
-    display: table;
-  }
+table tr,
+table th,
+table td {
+  border: none;
 }
 
-caption {
-  display: block;
+table th,
+table td {
+  text-align: left;
+  padding: $gutter/4;
+
+}
+
+table th {
+  text-transform: uppercase;
+}
+
+table tr td {
+  background: $gray-20;
+}
+
+table tr:nth-of-type(2n+2) td {
+  background: $white;
+}
+
+table.bt tbody td {
```

Comment:
When simulating a tablet view port, using the M/L/F options in the top bar in Pattern Labs, the table seems to be over running and causing a scrollbar to appear. I am noticing the “bt” class is added to the table elements only in mobile viewport. Does this also need to be applied to tablet?

![screen shot 2018-11-15 at 11 03 40 am](https://user-images.githubusercontent.com/541745/48568339-26b2ec80-e8cd-11e8-9d95-7edc2bb9a2a5.png)
![screen shot 2018-11-15 at 11 03 50 am](https://user-images.githubusercontent.com/541745/48568340-274b8300-e8cd-11e8-8d94-5bdf0b61e93b.png)
![screen shot 2018-11-15 at 11 03 59 am](https://user-images.githubusercontent.com/541745/48568341-274b8300-e8cd-11e8-9efc-8492846e351c.png)